d3ploy/demand_driven_deployment_inst.py

```diff
@@ -31,139 +31,141 @@ class DemandDrivenDeploymentInst(Institution):
     """
 
     facility_commod = ts.MapStringString(
-        doc="A map of facilities and each of their corresponding" +
-        " output commodities",
-        tooltip="Map of facilities and output commodities in the " +
-        "institution",
-        alias=['facility_commod', 'facility', 'commod'],
-        uilabel="Facility and Commodities"
+        doc="A map of facilities and each of their corresponding"
+        + " output commodities",
+        tooltip="Map of facilities and output commodities in the "
+        + "institution",
+        alias=["facility_commod", "facility", "commod"],
```

Comment:
What is the point of making this change everywhere? these two things ' and " are equivalent.

Reply:
I reverted all pep8 changes